app/assets/stylesheets/application.scss

```diff
@@ -566,3 +566,17 @@ table {
   max-height: 200px;
   padding: 4px;
 }
+
+.dropdown-menu a[disabled],
```

Comment:
Why all this CSS? `disabled` is not a valid attribute for a link at all - it does not prevent it from being clicked.